### ISSUE TITLE
Problem with behaviours

### ISSUE THREAD

**Issue by pharos-alexandria** [TEIC/CETEIcean#42]
If I add a behaviour like

```
"add": [
	["[place=margin]", ["<",">"]],
],
```
it behaves like expected: 

> <ἔξωθεν> 

But if I add html-tags to the behaviour like
```
"add": [
	["[place=margin]", ["<b>&lt;","&gt;</b>"]],
],
```
the output is

> **<>ἔξωθεν**

So, the angle bracket from the second array isn't any more output after the word.

Is this expected, i.e. do I need to write a function for such cases? Or is it a bug?

**Comment by hcayless** [comment]
Probably a bug. I'm investigating it now.

**Comment by hcayless** [comment]
Should be fixed with cc59e9c. I'll try to do an update release later.

**Comment by pharos-alexandria** [comment]
Works! Thank you!
